source/clear-linux/reference/faqs.rst

```diff
@@ -0,0 +1,200 @@
+.. _faqs:
+
+Community FAQs
```

Comment:
We should consider setting up the FAQ page to be a general FAQ (accommodate general questions officially answered in docs, as well as community questions).  Some questions here seem general enough we could pull them out into the general section e.g. frequency of updates, where to get software, how we do software... (and those can link to the supporting pages on site or in docs).

Reply:
> I think this should be surfaced to the top level nav in docs eg Get Started, Guides, Tutorials, FAQ,

I agree. I'll move it if no one else objects.

Reply:
No objections from me!

Reply:
I'm really glad that you're setting up an FAQ page. Our docs team talked about this last year. I think it will be a great reference for folks on the CL Forum. I agree with @puneetse that it should appear at the highest level so it's discoverable visually as well.

Reply:
I've pushed an update to move the document to top of tree.

Reply:
Looks good.